src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -974,6 +974,8 @@ private async Task PayoutBatchAsync(Balance[] balances)
                 return;
             }
 
+            if(web3Connection == null) InitializeWeb3(daemonEndpointConfig);
```

Comment:
can we change this to:

if(null == web3Connection && !string.IsNullOrEmpty(extraConfig.PrivateKey))